build

```diff
@@ -134,9 +134,6 @@ make_docs() {
         'yes')
             SCHEMA_MODULE='emqx_enterprise_schema'
             ;;
-        'no')
-            SCHEMA_MODULE='emqx_conf_schema'
-            ;;
     esac
```

Comment:
Nit: maybe we should bail out if this somehow resolves to something that's not `yes`?

Reply:
@zmstone what do say?

Reply:
sounds reasonable to be a bit defensive here.